### ISSUE TITLE
Move common addresses into a shared library

### ISSUE THREAD

**Issue by kargakis** [DefiLlama/DefiLlama-Adapters#4574]
Every adapter is redefining common addresses like WETH, USDC, etc. - it would be great to start moving these into a common library.

**Comment by g1nt0ki** [comment]
hi @kargakis I too have been thinking about it, slowly building a mapping here: [projects/helper/tokenMapping.js](https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/helper/tokenMapping.js#L18)

**Comment by kargakis** [comment]
@g1nt0ki it may make sense to start enforcing new adapters to use the library and slowly refactor the old ones. Ideally this would be automated somehow, otherwise just need to keep it in mind when reviewing PRs I guess.

**Comment by g1nt0ki** [comment]
yeah, I am not sure how to automate it, for now, have to enforce it via reviews

**Comment by sandbochs** [comment]
@g1nt0ki took a stab at automating it. check it out here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/5698

**Comment by g1nt0ki** [comment]
@0xmichalis I think this is a decent solution: https://github.com/DefiLlama/DefiLlama-Adapters/pull/6120

**Comment by 0xmichalis** [comment]
@g1nt0ki looks good!

**Comment by g1nt0ki** [comment]
finally merged: https://github.com/DefiLlama/DefiLlama-Adapters/pull/6120
